LicenseHeaderManager/PackageCommands/AddHeaderToAllFilesInProjectCommand.cs

```diff
@@ -14,17 +14,19 @@
 
 using System.Collections.Generic;
 using EnvDTE;
+using EnvDTE80;
 using LicenseHeaderManager.Headers;
 using LicenseHeaderManager.ReturnObjects;
 using LicenseHeaderManager.Utils;
+using Microsoft.VisualStudio.Shell;
 
 namespace LicenseHeaderManager.PackageCommands
 {
-  internal class AddLicenseHeaderToAllFilesCommand
+  internal class AddHeaderToAllFilesInProjectCommand
   {
     private LicenseHeaderReplacer licenseReplacer;
 
-    public AddLicenseHeaderToAllFilesCommand(LicenseHeaderReplacer licenseReplacer)
+    public AddHeaderToAllFilesInProjectCommand(LicenseHeaderReplacer licenseReplacer)
     {
       this.licenseReplacer = licenseReplacer;
```

Comment:
Thats a mistake by me, the private field should be named "_licenseReplacer" and the "this." should be removed. Just as information, as you used this naming scheme later on. If you have time you could clean it up, but i can do that myself later on. Just use the correct notation for the newly added code.